libs/action.js

```diff
@@ -145,6 +143,14 @@ let action = {
     return $(selector).waitForEnabled(waitTime);
   },
 
+  waitIsDisplayed(selector, waitTime = this.defaultWaitTime) {
+    try {
+      this.waitForVisible(selector, waitTime);
+    } catch (err) {
+      return false;
```

Comment:
Is it really necessary? it can be a different error, but always return false without the error message

Reply:
Yes, it is a custom method and really necessary to return false.